modules/rosa-hcp-sts-creating-a-cluster-cli.adoc

```diff
@@ -52,11 +52,11 @@ $ rosa create cluster --private --cluster-name=<cluster_name> \
     --sts --mode=auto --hosted-cp --subnet-ids=<private-subnet-id>
 ----
 
-** If you used variables like `OIDC_ID` and `SUBNET_IDS`, you can use those references when creating your cluster. For example, run the following command:
+** If you used the `OIDC_ID`, `SUBNET_IDS`, and `OPERATOR_ROLES_PREFIX` variables, you can use those references when creating your cluster. For example, run the following command:
```

Comment:
```suggestion
** If you used the `OIDC_ID`, `SUBNET_IDS`, and `OPERATOR_ROLES_PREFIX` variables to x, you can continue to use these variables when creating your cluster. For example, run the following command:
```

I added `x` in teh update as I cannot determine where these variables might have been previously used.